frame/dapps-staking/src/mock.rs

```diff
@@ -181,3 +180,9 @@ pub fn advance_to_era(n: EraIndex) {
         run_to_block(BLOCKS_PER_ERA * (n as BlockNumber - 1) + 1);
     }
 }
+
+/// adjust storage
+pub fn prepare_era_setup() {
+    DappsStaking::on_initialize(System::block_number());
+    advance_to_era(2);
```

Comment:
Can we omit this?

Let the test decide whether they want to start from era 1 or 2 or some other.